ftw/servicenavigation/upgrades/20200615183133_plone5_upgrade/upgrade.py

```diff
@@ -7,5 +7,9 @@ class Plone5Upgrade(UpgradeStep):
     """
 
     def __call__(self):
+        # replace plone.formwidget.contenttree using ftw.referencewidget
+        self.ensure_profile_installed('profile-ftw.referencewidget:default')
```

Comment:
You installing `ftw.referencewidget`s profile without installing it in the `setup.py`

Reply:
Oh, actually we do not need the upgrade step anymore, since it will now be a textfield...

Reply:
I removed the ensure installed in f726ba9